io.liri.BaseApp.yaml

```diff
@@ -0,0 +1,48 @@
+id: io.liri.BaseApp
```

Comment:
Please reformat file according to our guide: https://github.com/flathub/flathub/wiki/YAML-Style-Guide

Reply:
Done, thanks.